.lein-failures

```diff
@@ -0,0 +1 @@
+{"jackdaw.streams-test" ["KTable"]}
```

Comment:
Suspect this was not intended to be checked in?

Reply:
No, removed